Class-4/Practise/Practise_set_1/practise_set_1.js

```diff
@@ -0,0 +1,82 @@
+// console.log("Hello world")
+
+// console.log("This is coming first");
+// setTimeout(() => {
+//     console.log("This is coming at the last :( ")
+// }, 3000);
+
+// console.log("Now this is interesting, i am written at last but coming at second. Thanks to the asynchronus javascript");
+
+function generateRandomString(length) {
+    const characters = "mun";
+    let randomString = "";
+
+    for (let i = 0; i < length; i++) {
+        const randomIndex = Math.floor(Math.random() * characters.length);
+        randomString += characters.charAt(randomIndex);
+    }
+
+    return randomString;
+}
+
+// Promise Creation...
+
+const myPromis = new Promise((resolve, reject) => {
```

Comment:
I believe here also we had misspelled the word promise name, I hope it would be `myPromise` :wink: Though it's not a big deal but it will make you better in future

Reply:
Sorry vai for these mistakes. :(